tests/test_sdk.py

```diff
@@ -1,12 +1,12 @@
 import datetime as dt
-from unittest.mock import patch
 from unittest.mock import Mock
+from unittest.mock import patch
 
 import pandas as pd
 from mindsdb_sql import parse_sql
 
-import mindsdb_sdk
 from mindsdb_sdk.model import ModelVersion
+from testing import mindsdb_sdk
```

Comment:
can we also use just `import mindsdb_sdk` here and remove empty 'testing' module?

Reply:
That's actually possible but later on, as the project grows, your tests need to be updated and they'll need a separate `testing/` package.

Reply:
Although your tests depend on a third-party entity (`connect()` function) and it should be kept inside `testing/`, that's fine. I'll fix it.

Reply:
Just for information: `connect` is not third-party function, is a part of sdk, using connect is defined in sdk documentation.